de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/annotations/AnnotationManager.java

```diff
@@ -28,6 +29,9 @@
 //TODO move saved local selections affected by changes while editor is closed
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=203&issues=AWUe7ana8anAXINBEdcq&open=AWUe7ana8anAXINBEdcq)

[ISSUE_KEY]: AWUe7ana8anAXINBEdcq